src/pydrex/diagnostics.py

```diff
@@ -6,10 +6,10 @@
 
 $$
 a = \begin{bmatrix}
-    \cosφ\cosϕ - \cosθ\sinϕ\sinφ & \cosθ\cosϕ\sinφ + \cosφ\sinϕ & \sinφ\sinθ
-   -\sinφ\cosϕ - \cosθ\sinϕ\cosφ & \cosθ\cosϕ\cosφ - \sinφ\sinϕ & \cosφ\sinθ
-            \sinθ\sinϕ           &           -\sinθ\cosϕ        &   \cosθ
-    \end{bmatrix}
+    \\cosφ\\cosϕ - \\cosθ\\sinϕ\\sinφ & \\cosθ\\cosϕ\\sinφ + \\cosφ\\sinϕ & \\sinφ\\sinθ
+   -\\sinφ\\cosϕ - \\cosθ\\sinϕ\\cosφ & \\cosθ\\cosϕ\\cosφ - \\sinφ\\sinϕ & \\cosφ\\sinθ
+            \\sinθ\\sinϕ           &           -\\sinθ\\cosϕ        &   \\cosθ
+    \\end{bmatrix}
```

Comment:
This will ruin the LaTeX for generated docs (I'm thinking of using [pdoc](https://github.com/mitmproxy/pdoc) for that later, it's a lot simpler than sphinx). But I can maybe fix that later by doing `r"""text"""` instead of just `"""text"""`.

Reply:
Sure, and it should not be a problem to look for "\\" and replace by "\".